src/index.js

```diff
@@ -5,5 +5,19 @@ import Engine from './modules/engine';
 const game = new Game();
 const engine = new Engine(game.update, game.draw);
 
-engine.start();
-game.start();
+const startButton = document.querySelector('.start-button');
+const restartButton = document.querySelector('.restart-button');
+
+function startGame() {
+    engine.start();
```

Comment:
this can be called outside and still works the same, doesn't need to be called on every restart. we may later have to create a engine restart method so that the engine is fully reset.